Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [workspace]
 members = [
-  "uniffi",
+  "uniffi_bindgen",
+  "uniffi_build",
   "uniffi_macros",
+  "uniffi",
```

Comment:
At the risk of veering into bikeshed territory, I would be tempted to name this "uniffi_runtime" or similar to give a more direct indication that it contains the run-time support infra. We don't need to change it though, just sharing my reaction to the names here.

Reply:
I actually named it `uniffi_runtime` first, then I figured it would be sad if we didn't have any `uniffi` crate at all 😝 @tarikeshaq any opinion here?

Reply:
Haha, yeah, needing `uniffi` crate of *some* sort makes sense! I guess if/when we eventually move to a macro-based approach we'll also want that in a top-level `uniffi` crate if possible, so I'm happy to keep it this like.